src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
 #ifdef HAVE_SECCOMP
+static int syscall_filter_parse_one(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                ExecContext *c,
+                bool invert,
+                const char *t,
+                bool warn) {
+        int r;
+
+        if (*t == '@') {
+                const char * const *s;
+                static const char * const syscall_sets[] = {
+                        /* Default list */
+                        "@Default",
+                        "execve\0"
+                        "exit\0"
+                        "exit_group\0"
+                        "rt_sigreturn\0"
+                        "sigreturn\0",
+
+                        /* Need superuser capabilities */
+                        "@SysAdmin",
+                        "acct\0"
+                        "adjtimex\0"
+                        "bdflush\0"
+                        "bpf\0"
+                        "chown32\0"
+                        "chown\0"
+                        "chroot\0"
+                        "create_module\0"
+                        "delete_module\0"
+                        "fchown32\0"
+                        "fchown\0"
+                        "fchownat\0"
+                        "finit_module\0"
+                        "init_module\0"
+                        "ioperm\0"
+                        "iopl\0"
+                        "kexec_file_load\0"
+                        "kexec_load\0"
+                        "lchown32\0"
+                        "lchown\0"
+                        "mount\0"
+                        "nfsservctl\0"
+                        "oldumount\0"
+                        "pciconfig_read\0"
+                        "pciconfig_write\0"
+                        "pivot_root\0"
+                        "quotactl\0"
+                        "reboot\0"
+                        "seccomp\0"
+                        "setdomainname\0"
+                        "setfsgid32\0"
+                        "setfsgid\0"
+                        "setfsuid32\0"
+                        "setfsuid\0"
+                        "setgid32\0"
+                        "setgid\0"
+                        "setgroups32\0"
+                        "setgroups\0"
+                        "sethostname\0"
+                        "setregid32\0"
+                        "setregid\0"
+                        "setresgid32\0"
+                        "setresgid\0"
+                        "setresuid32\0"
+                        "setresuid\0"
+                        "setreuid32\0"
+                        "setreuid\0"
+                        "settimeofday\0"
+                        "setuid32\0"
+                        "setuid\0"
+                        "stime\0"
+                        "swapoff\0"
+                        "swapon\0"
+                        "sysctl\0"
+                        "syslog\0"
+                        "umount2\0"
+                        "umount\0"
```

Comment:
One of the main issues with caps is that there's no separate capability for controlling mount/umount stuff. I am very sure that we should emphesize this when defining these groups, and add a separate group for "mount" and "umount", probably just called `@mount`.